CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+2.0.0
+-----
+
+* Upgraded to Django 2.2.
+* Dropped support for Django 1.11 and versions below it.
+* Dropped Python 2.7 support.
+
+
 1.x.x
 -----
```

Comment:
I think this `1.x.x` heading should go away, and the Markdown bump can be listed under 2.0.0 release, since there is no version between these two.